src/crud/passage.ts

```diff
@@ -64,21 +64,22 @@ export function parseRef(a: Passage) {
     }
   }
 }
-
-/* build the passage ref = book + reference */
-export function passageReference(passage: Passage, bookData: BookName[] = []) {
+export function passageBook(passage: Passage, bookData: BookName[] = []) {
   const attr = passage.attributes;
-  if (!attr) return '';
-  let book = attr.book || '';
+  let book = attr?.book || '';
```

Comment:
according the the Passage.ts in the model folder, every passage has to have a book which is always a string. Is the API now returning passages without a book in the passage?

Reply:
It would be good to do this anyway so I'll just make this a comment.